scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -80,7 +80,7 @@ workflow GvsJointVariantCalling {
             SNP_VQSR_CLASSIC_max_gaussians_override = SNP_VQSR_CLASSIC_max_gaussians_override,
             SNP_VQSR_CLASSIC_mem_gb_override = SNP_VQSR_CLASSIC_mem_gb_override,
             drop_state = drop_state,
-            is_beta_user = is_beta_user,
+            is_beta_user = tighter_gcp_quotas,
```

Comment:
hmm might be nice to use the same terminology on both sides of the `=`

Reply:
I agree.  But really, the main motivator was to present a better user-facing variable to beta users.  I just got the feeling that if we exposed it as is_beta_customer it's the sort of thing that might get played with much more than if we exposed it as tighter_gcp_quotas.

But I didn't think that justified changing terminology the whole way down the stack.  Maybe we want to in a different ticket?

Reply:
ok makes sense